### PR TITLE
feat:core: align underscores

### PR DESCRIPTION
**Pull request by nmimran99**
feat: align underscores for contracts

Align the usage of underscores according to variables and functions naming convention.
https://www.notion.so/prepo/Smart-Contract-Handbook-73d78c1d6eec46e2adb225d7162b48c4?pvs=4#3f4e9cfda0354a00aa4e375e08c18d6f

### CODE REVIEW

**Review by ramenforbreakfast — CHANGES_REQUESTED** [review]
Missing changes for `PrePOMarket.sol` which also has missing underscores

File: apps/smart-contracts/core/contracts/PrePOMarketFactory.sol [hunk 1]
```diff
@@ -25,106 +25,103 @@ contract PrePOMarketFactory is
...
 
-  function isValidCollateral(address _collateral)
+  function isValidCollateral(address collateral)
     external
     view
     override
     returns (bool)
   {
-    return validCollateral[_collateral];
+    return validCollateral[collateral];
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/contracts/PrePOMarketFactory.sol:34]: the mappings are also private variables, so they should have underscores too

**Review by XavierEkkel — APPROVED** [review]
LGTM after comment and if all tests pass (please add PR title prefix)

**Review by ramenforbreakfast — APPROVED** [review]
LGTM, except `validCollateral` and `deployedMarkets` need to be underscored in `PrePOMarketFactory.sol` and the below change.

File: apps/smart-contracts/core/contracts/interfaces/IPrePOMarket.sol [hunk 2]
```diff
@@ -76,13 +76,13 @@ interface IPrePOMarket {
    *
    * After the market has ended, users can redeem any amount of
    * Long/Short tokens for Collateral.
-   * @param _longAmount Amount of Long tokens to redeem
-   * @param _shortAmount Amount of Short tokens to redeem
+   * @param longAmount Amount of Long tokens to redeem
```
Comment (ramenforbreakfast) [inline apps/smart-contracts/core/contracts/interfaces/IPrePOMarket.sol:79]: `_amount` in `mint` needs to be `amount`

**Pull request merged**